ir/instr.cpp

```diff
@@ -254,6 +254,7 @@ void UnaryOp::print(ostream &os) const {
   const char *str = nullptr;
   switch (op) {
   case BitReverse:  str = "bitreverse "; break;
+  case BSwap:       str = "bswap "; break;
```

Comment:
The PR title is misleading then?

Reply:
yes, i was about to change that; you were too quick :)